### PR TITLE
Fix incorrect support for long CommandLine strings in procmon plugin

### PR DESCRIPTION
**Pull request by disaykin**
Previous implementation:
- does not work on x86 arch (uses incorrest offset for _UNICODE_STRING->Buffer)
- does not convert UTF-16 to UTF-8 (copies only non-zero bytes instead)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	ignore = dirty
 [submodule "libvmi"]
 	path = libvmi
-	url = https://github.com/tklengyel/libvmi
+	url = https://github.com/libvmi/libvmi
```
Comment (tklengyel) [inline .gitmodules:7]: Please don't change the url here.

Comment (tklengyel) [reply .gitmodules:7]: I've pushed the latest master to `tklengyel/libvmi` you should be able to update the submodule to that.

Comment (disaykin) [reply .gitmodules]: I just didn't know how to ask you to do this. Done.

Comment (tklengyel) [reply .gitmodules]: No worries

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Pull request merged**